hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1929,6 +1929,10 @@ public List<String> getColumnsEnabledForBloomFilterIndex() {
     return getMetadataConfig().getColumnsEnabledForBloomFilterIndex();
   }
 
+  public boolean shouldEnableBasePathForPartitions() {
```

Comment:
if we go w/ my suggestion below for the naming, lets name this method

`shouldEnableBasePathOverrideForMetadataWrites()`